pkg/selector/eks.go

```diff
@@ -0,0 +1,124 @@
+package selector
+
+import (
+	"archive/zip"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+const (
+	eksAMIRepoURL               = "https://github.com/awslabs/amazon-eks-ami"
+	eksFallbackLatestAMIVersion = "v20210125"
```

Comment:
np: would be neat to fetch EKS AMI based on k8s major/minor versions per this [table](https://docs.aws.amazon.com/eks/latest/userguide/eks-linux-ami-versions.html)

Reply:
I think we'd still need the date though and the instance types supported don't change across major.minor k8s versions. So doing that would only require users to input more information not less, but we probably should support the full version as an option of input. We can just pull off the date at the end to actually do the lookup.